### PR TITLE
fix: add makendrang

### LINKED ISSUE #1316: Makendran G
image

### PR DESCRIPTION
**Pull request by MakendranG**
resolve #1316

resolves #issueNumber

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/makendrang.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+![image](https://user-images.githubusercontent.com/74320000/185746011-a5c06825-183d-4dc3-abd5-bd057098381e.png)
```
Comment (Panquesito7) [inline profiles/makendrang.json:1]: Please remove this line.
```suggestion
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by MakendranG** [comment]
Suggested changes have been done.

File: profiles/makendrang.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (schmelto) [inline profiles/makendrang.json:1]:
```suggestion
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🚀


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Pull request merged**